src/App.js

```diff
@@ -1,22 +1,39 @@
 import "./App.css";
-import { Switch, Route } from "react-router-dom";
+import { Route, NavLink, Switch } from "react-router-dom";
+import Profile from "./components/Profile";
+import Places from "./components/Places";
+import Map from "./components/Map";
 
 function App() {
   return (
     <div className="App">
-      <header class="header">Awesome Header</header>
-      <main>Awesome Content</main>
-      <Switch>
-        <Route path="/places"></Route>
-      </Switch>
-      <Switch>
-        <Route path="/profile"></Route>
-      </Switch>
-
-      <Switch>
-        <Route exact path="/"></Route>
-      </Switch>
-      <footer>Awesome Footer</footer>
+      <header className="header">
+        <Switch>
+          <Route path="/map">
+            <Map />
+          </Route>
+          <Route path="/places">
+            <Places />
+          </Route>
+          <Route path="/profile">
+            <Profile />
+          </Route>
+        </Switch>
```

Comment:
I think it's best practice to have a home path ("/"). If let's say /map is used as your home or main page, you can use redirect:
         `<Redirect from="/" to="/map" />`  
or
`        <Route exact path="/">
          <Redirect to="/map" />
        </Route>
`

Reply:
true, I need to include the home path too. just wasn't sure what my main page will be.